src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -318,7 +318,7 @@ nuget pack foo.nuspec -Version 2.1.0</value>
     <value>Pushes a package to the server and publishes it.</value>
   </data>
   <data name="PushCommandSourceDescription" xml:space="preserve">
-    <value>Specifies the server URL. If not specified, nuget.org is used unless DefaultPushSource config value is set in the NuGet config file.</value>
+    <value>Specifies the server URL. NuGet identifies a UNC or local folder source and simply copies the file there instead of pushing it using HTTP. Also, starting with NuGet 3.4.2, this is a mandatory parameter unless the NuGet.Config file specifies a DefaultPushSource value.</value>
```

Comment:
The "starting with NuGet 3.4.2" part is appropriate for docs where the version being used is unknown. But this is being checked in to code where we know this is the behaviour, so I would start the last sentence with "This is a mandatory ....".

Reply:
Thanks, that’s a very good point! I adjusted the text to avoid mentioning the version.